### ISSUE TITLE
Dgraph: uid-set test never finishes, loops forever

### ISSUE THREAD

**Issue by manishrjain** [jepsen-io/jepsen#307]
When running Jepsen test with workload `uid-set`:

```
# lein run test --local-binary /gobin/dgraph --force-download --nemesis move-tablet --upsert-schema --time-limit 300 --concurrency 2n --nemes-interval 5 --workload uid-set --test-count 10
```

The reads no longer time out. But, the test never finishes. It keeps on spitting:

```
2019-02-01 23:22:16,808{GMT}	INFO	[jepsen worker 1] jepsen.dgraph.set: Forcing conflict by deleting ...
```

Looks like these conflicts don't happen, and the workers just keep looping over them forever. Might be a logic based on an old behavior of Dgraph which no longer happens.

**Comment by aphyr** [comment]
If I recall correctly, we need to force a conflict here in order to ensure we do a recent read--if we didn't, we could observe past state and possibly observe "missing" values which were actually present. It's been a while since I've been in this code though, so... take that with a grain of salt. :)

**Comment by manishrjain** [comment]
With the updated design, every read takes in a new timestamp from Zero, and the Alpha would ensure that it has all the data needed to serve reads >= that timestamp. Therefore, every read must be the latest read (unless the test is doing something weirder).

**Comment by aphyr** [comment]
Oh, yeah, if Dgraph is supposed to guarantee linearizable reads now, then I think you're fine to remove this. This was just a safety mechanism to let us do reads in a system that only provided SI, without relying on recency guarantees. :)

**Comment by manishrjain** [comment]
Removed that, so the test does not hang forever. But, I realize the test does not consider txn aborts. Aborts are considered `:ok` for some reason, causing the test to fail.

**Comment by mkcp** [comment]
Resolved the aborts in 2510feb69f852119eb959c803f22ceb8846ae19f
